test/commands/distribute/releases/edit-test.ts

```diff
@@ -148,7 +141,21 @@ describe("distribute releases edit command", async () => {
       const executionScope = _.flow(setupReleaseDetailsResponse, setupUpdateReleaseServiceUnavailableResponse)(Nock(fakeHost));
 
       // Act
-      const command = new EditReleaseCommand(getCommandArgs([releaseIdOption, fakeReleaseId, "disabled"]));
+      const command = new EditReleaseCommand(getCommandArgs([releaseIdOption, fakeReleaseId, "--state disabled"]));
+      const result = await command.execute();
+
+      // Assert
+      expect((result as CommandFailedResult).errorMessage).to.eql(expectedErrorMessage);
+      testCommandFailure(executionScope);
+    });
+
+    it("if file and release notes both provided", async () => {
+      // Arrange
+      const expectedErrorMessage = "'--release-notes' and '--release-notes-file' switches are mutually exclusive";
+      const executionScope = _.flow(setupReleaseDetailsResponse, setupUpdateReleaseServiceUnavailableResponse)(Nock(fakeHost));
+
+      // Act
+      const command = new EditReleaseCommand(getCommandArgs([releaseIdOption, fakeReleaseId, "--relese-notes anytext --release-notes-file some/fake/path"]));
```

Comment:
You should investigate why this test passes. The parameter is misspelled: `--relese-notes`. That means it shouldn't throw the expected error.